core/contracts/FixedPoint.sol

```diff
@@ -21,6 +21,11 @@ library FixedPoint {
         return Unsigned(a.mul(FP_SCALING_FACTOR));
     }
 
+    /** @dev Whether `a` is equal to `b`. */
+    function isEqual(Unsigned memory a, Unsigned memory b) internal pure returns (bool) {
```

Comment:
Would you mind adding a quick test for this where the other fixed point functions are tested?

Reply:
sure. Will expand the tests when the next PR to expand the `fixedpoint` lib.